File: 344-reverse-string/reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        // vector<char> s2;
        // int n = s.size();
        // for(int i=n-1;i>=0;i--){
        //     s2.push_back(s[i]);
        // }
        // for(int i=0;i<n;i++){
        //     s[i] = s2[i];
        // }
        ////////////////////
        reverse(begin(s),end(s));

    }
};

File: 344-reverse-string/reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        if(len(s) > 1):
            left = 0
            right = len(s) - 1

            while(left < right):
                s[left] , s[right] = s[right] , s[left]
                left = left + 1
                right = right - 1

        